support/cas-server-support-inwebo-mfa/src/main/java/org/apereo/cas/support/inwebo/web/flow/actions/SuccessAction.java

```diff
@@ -0,0 +1,19 @@
+package org.apereo.cas.support.inwebo.web.flow.actions;
+
+import org.springframework.webflow.action.AbstractAction;
+import org.springframework.webflow.execution.Event;
+import org.springframework.webflow.execution.RequestContext;
+
+/**
+ * A simple web action for success.
+ *
+ * @author Jerome LELEU
+ * @since 6.4.0
+ */
+public class SuccessAction extends AbstractAction {
```

Comment:
Should be unnecessary. Use `StaticEventExecutionAction`

Reply:
Yes. Done alread.